app/Http/Controllers/IssueController.php

```diff
@@ -7,10 +7,24 @@
 
 class IssueController extends Controller
 {
+    private const ACCEPTABLE_SORT_FIELDS = ['title', 'repoName', 'createdAt'];
+    private $sort;
+
+    public function __construct()
```

Comment:
Could we please remove the `__construct()`? At the top of the `invoke` method, we could maybe use something like this:

```php
$sortBy = in_array(request('sort'), ['title', 'repoName', 'createdAt'], true)
    ? request('sort')
    : null;
```